audiomentations/core/utils.py

```diff
@@ -16,31 +16,36 @@
     ".opus",
     ".wav",
 )
-
-
-def get_file_paths(
-    root_path: Union[str, Path],
-    filename_endings=AUDIO_FILENAME_ENDINGS,
+    
+def find_audio_files_in_paths(
+    paths: Union[List[Path], List[str], Path, str],
+    filename_endings=SUPPORTED_EXTENSIONS,
```

Comment:
`E   NameError: name 'SUPPORTED_EXTENSIONS' is not defined`

Reply:
Fixed.